### DISCUSSION TITLE
Filter for Comments more recents than specified date

### DISCUSSION THREAD

**Discussion by NatNgs** [ThioJoe/YT-Spammer-Purge#423] [Ideas]
Like the mode to scan a certain number of recent comment from the whole channel, I would suggest to add a mode (or an option / filter...) to set the limit as a date (check all comments more recent than given date) instead of a number of comments.

My idea, is that I know when I did run the check last time, but I do not know how much new comments there was on my/selected channel to check

**Comment by ThioJoe** [comment]
The reason you probably wouldn't want to do that is because if you scan a comment once, then it gets a reply, you would never scan that reply. You have to always scan the top level comment to see the replies, there is no way to get a list of the most recent replies separate from the top level comment.
